function func() {
   return new Promise((resolve,reject)=>{
       setTimeout(()=>{
         const error = false;
         if(error)
         {
            console.log("Some error is there");
            reject(50);
         }
         else{
            console.log("No error");
            resolve(60);
         }
       },2000)
   })
}
var a=1;
console.log("Hellpo");
func().then(data=>a=data).catch(data=>console.log(data)) // It gurantees me to perform this task only when the promise function is executed
console.log(a);
console.log("World");
